python/cudf_polars/tests/test_config.py

```diff
@@ -58,6 +58,18 @@ def test_invalid_memory_resource_raises(mr):
         q.collect(engine=pl.GPUEngine(memory_resource=mr))
 
 
+@pytest.mark.parametrize("disable_managed_memory", ["1", "0"])
+def test_cudf_polars_enable_disable_managed_memory(monkeypatch, disable_managed_memory):
```

Comment:
I think you might need to skip code coverage on some a few of the lines in `callback.py` or expand the test.